src/publish.proj

```diff
@@ -37,9 +37,12 @@
 
   <!-- Collect test archives for pushing to transfer feed -->
   <ItemGroup>
-    <TestArchives Include="$(TestArchivesRoot)**/*.zip" />
-    <TestArchives Include="$(AnyOSTestArchivesRoot)**/*.zip" />
-    <TestArchives Condition="'$(TargetsUnix)' == 'true'" Include="$(UnixTestArchivesRoot)**/*.zip" />
+    <TestArchive Include="$(TestArchivesRoot)**/*.zip" />
```

Comment:
I don't see the point of moving from plural to singular but I'd point out **/*.zip is more than one zip.

Reply:
Based on Aaron's comments above, I think this is a convention, but I am not familiar with it.

Reply:
The reason in this case is that an MSBuild Item is by definition a part of a collection and each Item tag represents a unique instance in that collection. Wildcards can be used as @josalem is doing here and so there are multiple files in the statement, but in MSBuild each resolved file from the wildcard gets a unique Item instance.

It is definitely style not functional.